script/cibuild

```diff
@@ -10,7 +10,13 @@ ruby -v
 echo "hostname: $(hostname)"
 echo "pwd: $(pwd)"
 git --version
+echo "node: $(node --version)"
+echo "npm: $(npm --version)"
 
 script/bootstrap
 
 bundle exec rake test
+
+find ./_site -type d \( -name assets -o -name _javascript -o -wholename downloads/es_ES/github-git-cheat-sheet/index.html \) -prune -o -name "*.html" -print >> ./script/list-of-files.txt
```

Comment:
The first two parts of this `-prune` work fine, omitting files from the assets file and the _javascript file. BUT, I would really like to include specific cheat sheets, as well. Any suggestions on how to do this?